src/packages/src/xroad/ubuntu/generic/xroad-securityserver-fi.postinst

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ "$1" = configure ]; then
+  invoke-rc.d --quiet xroad-proxy try-restart || true
+  invoke-rc.d --quiet xroad-signer try-restart || true
+  invoke-rc.d --quiet xroad-addon-messagelog try-restart || true
```

Comment:
messagelog is not a separate service (xroad-proxy plugin), no need to restart it.

Reply:
Fixed this to all